docs/release-process.md

```diff
@@ -172,14 +177,14 @@ git checkout -b kops_images_${VERSION}
 cd k8s.gcr.io/images/k8s-staging-kops
 echo "" >> images.yaml
 echo "# ${VERSION}" >> images.yaml
-k8s-container-image-promoter --snapshot gcr.io/k8s-staging-kops --snapshot-tag ${VERSION} >> images.yaml
+k8s-container-image-promoter run --snapshot gcr.io/k8s-staging-kops --snapshot-tag ${VERSION} >> images.yaml
 ```
 
 You can dry-run the promotion with 
 
 ```
 cd ${GOPATH}/src/k8s.io/k8s.io
-k8s-container-image-promoter --thin-manifest-dir k8s.gcr.io
+k8s-container-image-promoter run --thin-manifest-dir k8s.gcr.io
```

Comment:
My attempts to dry-run the promotion all complained about my permissions, so this is not entirely accurate.